lib/testing/src/lib/process-services-cloud/actions/process.services.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ApiService } from '../../core/actions/api.service';
+import { ProcessDefinitionsService } from './process-definitions.service';
+import { ProcessInstancesService } from './process-instances.service';
+import { QueryService } from '../../core/actions/identity/query.service';
+import { TasksService } from './tasks.service';
+import { StringUtil } from '../../core/utils/string.util';
+
+export class ProcessServices {
```

Comment:
I think we already have a similar file https://github.com/Alfresco/alfresco-apps/blob/489933c4d631dba878f3c6871e051aec3b4b9107/apps%2Fadmin-ee-e2e%2Fsrc%2Fapi%2Fdeployment%2Fprocess.api.ts 

why don't we move. that to adf testing?

Reply:
Did some changes accordingly